### PR TITLE
Ci integration and fixing lint errors

### PR DESCRIPTION
**Pull request by MeenuV**
#6

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+requests
```
Comment (AliceYuan) [inline requirements.txt]: think there's a lot more than this:
wikipedia
flask

there's other ones but I'm forgetting them

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
+exclude = client/__init__.py
```
Comment (AliceYuan) [inline setup.cfg]: why exclude __init__?

File: requirements.txt (hunk 1 above)
Comment (MeenuV) [inline requirements.txt]: Thanks for reminding me of wikipedia. Flask is being done seperately as I specify a flask env.

File: setup.cfg (hunk 2 above)
Comment (MeenuV) [inline setup.cfg]: It's just a blank file that has nothing in it

File: requirements.txt (hunk 1 above)
Comment (AliceYuan) [inline requirements.txt]: httplib
urlparse
HTMLParser.
not sure if all are installs or standard lib

File: setup.cfg (hunk 2 above)
Comment (AliceYuan) [inline setup.cfg]: I would just delete init if that's the case, cause then we have to remember to remove exclude
can also just put 
pass

File: requirements.txt (hunk 1 above)
Comment (MeenuV) [inline requirements.txt]: Those all seem to be standard libs

File: setup.cfg (hunk 2 above)
Comment (MeenuV) [inline setup.cfg]: `pass`ed

Comment (AliceYuan) [inline setup.cfg]: don't forget to remove the exclude :p

**Comment by AliceYuan** [comment]
r+

**Pull request merged**